src/schemaModels/Work/Experience.js

```diff
@@ -0,0 +1,30 @@
+class Experience {
+  constructor(company, position, website, startDate, endDate, summary, highlights) {
+    this.company = company
+    this.position = position
+    this.website = website
+    this.startDate = startDate
+    this.endDate = endDate
+    this.summary = summary
+    this.highlights = highlights
+  }
+
```

Comment:
I haven't seen this type of constructor method before. I'm not sure but it seems a bit redundant since you aren't actually holding these values as state. Perhaps could use functional components and refer to the data points as props being passed down from the parent component